examples/classification.pct.py

```diff
@@ -150,7 +145,7 @@
 L_inv = jsp.linalg.solve_triangular(L, I(D.n), lower=True)
 H_inv = jsp.linalg.solve_triangular(L.T, L_inv, lower=False)
 
-laplace_approximation = dx.MultivariateNormalFullCovariance(f_map_estimate, H_inv)
+laplace_approximation = numpyro.distributions.MultivariateNormal(f_map_estimate, H_inv)
```

Comment:
Would be nice to import `numpyro.distributions` as `npd` throughout this notebook.